### DISCUSSION TITLE
Errors converting tif to pdf

### DISCUSSION THREAD

**Discussion by ShahMV** [ImageMagick/ImageMagick#2588] [Help]
When trying to convert the attached tif files to pdf, I get the following errors:

magick multipleBarcodesWithText-200dpi-fax-8.5by11.tif multipleBarcodesWithText-200dpi-fax-8.5by11.pdf
magick: Unknown field with tag 317 (0x13d) encountered. `TIFFReadDirectory' @ warning/tiff.c/TIFFWarnings/1044.
magick: ASCII value for tag "Software" contains null byte in value; value incorrectly truncated during reading due to implementation limitations. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.
magick: Incompatible type for "RichTIFFIPTC"; tag ignored. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.
magick: ASCII value for tag "Software" contains null byte in value; value incorrectly truncated during reading due to implementation limitations. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.
magick: Incompatible type for "RichTIFFIPTC"; tag ignored. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.

magick multipleBarcodesWithText-200dpi-huffman-grayscale.tif multipleBarcodesWithText-200dpi-huffman-grayscale.pdf
magick: Unknown field with tag 317 (0x13d) encountered. `TIFFReadDirectory' @ warning/tiff.c/TIFFWarnings/1044.
magick: ASCII value for tag "Software" contains null byte in value; value incorrectly truncated during reading due to implementation limitations. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.
magick: ASCII value for tag "Software" contains null byte in value; value incorrectly truncated during reading due to implementation limitations. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.

magick multipleBarcodesWithText-200dpi-lzw-grayscale.tif multipleBarcodesWithText-200dpi-lzw-grayscale.pdf
magick: ASCII value for tag "Software" contains null byte in value; value incorrectly truncated during reading due to implementation limitations. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.
magick: Unknown field with tag 317 (0x13d) encountered. `TIFFReadDirectory' @ warning/tiff.c/TIFFWarnings/1044.

magick BarcodesAndText-300dpi-Fax-BW.tif BarcodesAndText-300dpi-Fax-BW.pdf
magick: Unknown field with tag 317 (0x13d) encountered. `TIFFReadDirectory' @ warning/tiff.c/TIFFWarnings/1044.
magick: ASCII value for tag "Software" contains null byte in value; value incorrectly truncated during reading due to implementation limitations. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.
magick: ASCII value for tag "Software" contains null byte in value; value incorrectly truncated during reading due to implementation limitations. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.

My imagemagick version:

Version: ImageMagick 7.0.10-29 Q16 x64 2020-09-05 http://www.imagemagick.org
Copyright: Copyright (C) 1999-2018 ImageMagick Studio LLC
License: http://www.imagemagick.org/script/license.php
Visual C++: 192729111
Features: Cipher DPC HDRI Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg lcms lqr lzma openexr pangocairo png ps raw rsvg tiff webp xml zlib

Any helps or suggestions would be welcomed.  Thank you.

[Tif2PDFErrors.zip](https://github.com/ImageMagick/ImageMagick/files/5234899/Tif2PDFErrors.zip)

**Comment by snibgo** [comment]
In that list, eg...
```
TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/1044.
```
... I can see no errors, only warnings. "-quiet" will supress warnings.

**Comment by ShahMV** [comment]
Ok. fair point.  However when I try to view the output pdf they all give an error saying 'Insufficient data for an image'
[OutputPDF.zip](https://github.com/ImageMagick/ImageMagick/files/5236388/OutputPDF.zip)

**Comment by fmw42** [comment]
Works fine for me with 

`magick -quiet BarcodesAndText-300dpi-Fax-BW.tif BarcodesAndText-300dpi-Fax-BW.pdf`

on IM 7.0.11.29 Q16 Mac OSX Sierra with no error messages. -quiet has suppressed the warnings. I get an output PDF that looks reasonable.
